### PR TITLE
UI change

### PR DESCRIPTION
**Pull request by NabajitS**
The updated modal looks like this
changed

### CODE REVIEW

**Comment by NabajitS** [comment]
issue #275.
I couldn't figure out why the files `.github/ISSUE_TEMPLATE/bug_report.md` and `.github/ISSUE_TEMPLATE/feature_request.md` keep getting added to my changed files.

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 .eslintcache
+/.github
```
Comment (idanasulinmemphis) [inline .gitignore:8]: please remove this one from the PR

File: src/domain/overview/style.scss [hunk 2]
```diff
@@ -4,21 +4,25 @@
     padding: 1vw;
```
Comment (idanasulinmemphis) [inline src/domain/overview/style.scss:4]: youe are probably using some linter which adding empty lines, no new css settings were committed so please remove it also from the PR

Comment (NabajitS) [reply src/domain/overview/style.scss:4]: I did add a new line to this file. It's at line 176. I saw the spaces added by my linter only after making the PR.

File: .gitignore (hunk 1 above)
Comment (NabajitS) [reply .gitignore]: Done

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**